engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -217,7 +217,41 @@ private void initRenderGraph() {
         NodeFactory nodeFactory = new NodeFactory(context);
         RenderGraph renderGraph = new RenderGraph();
 
+        FBOConfig sceneOpaqueFboConfig = displayResolutionDependentFBOs.getFboConfig(READONLY_GBUFFER);
+
         // ShadowMap generation
+        addShadowMapNodes(renderGraph,nodeFactory);
+        // (i.e. water) reflection generation
+        addWaterReflectionNodes(renderGraph,nodeFactory);
+        // sky rendering
+        addSkyNodes(renderGraph,nodeFactory,sceneOpaqueFboConfig);
+
+        // world rendering
+        addWorldNodes(renderGraph,nodeFactory);
+
+        // TODO: remove this, including associated method in the RenderSystem interface
+        Node firstPersonViewNode = nodeFactory.createInstance(FirstPersonViewNode.class);
+        renderGraph.addNode(firstPersonViewNode, "firstPersonViewNode");
+
+        // lighting
+        addLightingNodes(renderGraph,nodeFactory);
+
+        // TODO: consider having a none-rendering node for FBO.attachDepthBufferTo() methods
+
+        // 3d-based decorations (versus purely 2d, post-production effects)
+        add3dDecorationNodes(renderGraph,nodeFactory);
+
+        
+        // Pre-post-processing, just one more interaction with 3D data (semi-transparent objects, in SimpleBlendMaterialsNode)
+        // and then it's 2D post-processing all the way to the image shown on the display.
+        addPostProcessingNodes(renderGraph,nodeFactory,sceneOpaqueFboConfig);
```

Comment:
For this and other new methods below, please order them in the same order as they appear in initRenderGraph().